third_party/wpantund/Makefile.am

```diff
@@ -40,6 +40,7 @@ override CFLAGS                   := $(filter-out -pedantic-errors -Werror,$(CFL
 override CXXFLAGS                 := $(filter-out -Werror,$(CXXFLAGS))
 
 libwpanctl_la_SOURCES              = \
+	repo/src/util/string-utils.c     \
```

Comment:
nit: replace tab with spaces

Reply:
Thank you! I will pay more attention on such kind of issues.